zap/src/main/java/org/parosproxy/paros/extension/manualrequest/http/impl/HttpPanelSender.java

```diff
@@ -247,7 +253,7 @@ private JToggleButton getButtonUseTrackingSessionState() {
                     new JToggleButton(
                             new ImageIcon(
                                     HttpPanelSender.class.getResource(
-                                            "/resource/icon/fugue/cookie.png"))); // Cookie
+                                            "/resource/icon/fugue/globe-green.png")));
```

Comment:
What's the source/license of this icon?

Reply:
From fugue icons, under Creative Commons 3.0 (the other icons seem to come from the same icon set so I assume attribution was made)

Reply:
Yeah, just checking :)